optuna/visualization.py

```diff
@@ -92,6 +93,74 @@ def _get_intermediate_plot(study):
     return figure
 
 
+def plot_optimization_history(study):
+    # type: (Study) -> None
+    """Inside Jupyter notebook, plot optimization history of all trials in a study.
```

Comment:
This method is not necessarily invoked in Jupyter notebook. How about just mentioning `Plot optimization history of all trials in a study.`?